multi-agent/run/scripts/migrate_builtin_system.py

```
@@ -0,0 +1,445 @@
+                registry = ElementRegistry()
+                registry.auto_discover()
+                field_enc = ResourceFieldEncryption(registry, cipher)
+                _, sensitive = field_enc.scan_schema_hints(category, type_key)
+                model_cls = registry.get_schema(ResourceCategory(category), type_key)
+                keys = sensitive | set(getattr(model_cls, "ENCRYPTED_FIELDS", ()))
+            except (KeyError, ImportError, ValueError) as exc:
+                raise RuntimeError(
+                    f"Cannot resolve sensitive keys for {category}/{type_key} "
+                    f"while encryption is enabled — aborting to prevent "
+                    f"plaintext migration: {exc}"
+                ) from exc
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)